### PR TITLE
Close #22173: Patrol path tiles change colour to show selection.

### LINKED ISSUE #22173: new patrol area "cursor" hidden by existing patrol area highlighting
### Operating System

Ubuntu Linux 22.04.4

### OpenRCT2 build

2fbfaf2 develop, Linux AppImage

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

The white tiles "cursor" that is used to set a new patrol area is not shown on tiles that already have another staff's patrol area set.

https://github.com/OpenRCT2/OpenRCT2/assets/29800630/171d903a-160b-417e-a9dd-7413801fcc4c

That makes detailed patrol area management in busy parts of a park quite tricky.

### Steps to reproduce

- set patrol area of staff1
- try to set patrol area of staff2 in the same place
- white tile highlighting disappears on existing staff1 patrol area

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by mrmbernardi**
Fixes #22173

screenshot-2024-06-27_19:21:34

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Nice idea! Looks good to me.

**Review by AaronVanGeffen — APPROVED** [review]

File: distribution/changelog.txt [hunk 1]
```diff
@@ -42,6 +42,7 @@
 - Fix: [#22121] Some news items in the “Recent Messages” window have the wrong text colour.
 - Fix: [#22152] [Plugin] Negative signed integers are truncated.
 - Fix: [#22161] Using arrow keys in textboxes crashes the game.
+- Fix: [#22173] Patrol path selection not visible over existing patrol paths.
```
Comment (AaronVanGeffen) [inline distribution/changelog.txt:45]: Probably more of a `Change` than a `Fix`, right?

Comment (ZehMatt) [reply distribution/changelog.txt:45]: I think so too.

Comment (mrmbernardi) [reply distribution/changelog.txt]: I've updated it

**Comment by cheweytoo** [comment]
Works a treat - thanks a lot!

**Pull request merged**